service-print/src/main/java/org/oskari/print/util/StyleUtil.java

```diff
@@ -86,9 +85,10 @@ public static PDPrintStyle getPolygonStyle (JSONObject oskariStyle, PDResources
 
     public static PDPrintStyle getPointStyle (JSONObject oskariStyle, PDDocument doc) throws IOException {
         PDPrintStyle style = new PDPrintStyle();
-        JSONObject image = JSONHelper.getJSONObject(oskariStyle, "image");
+        JSONObject image = JSONHelper.optJSONObject(oskariStyle, "image", () -> new JSONObject());
         int shape = image.optInt("shape", 5); // External icons not supported
-        String fillColor = JSONHelper.optString(JSONHelper.getJSONObject(image, "fill"), "color");
+        JSONObject imageFill = JSONHelper.optJSONObject(image, "fill", () -> new JSONObject());
+        String fillColor = JSONHelper.optString(imageFill, "color");
         style.setFillColor(ColorUtil.parseColor(fillColor));
         int size = image.optInt("size", 3);
         style.setIcon(getIcon(doc, shape, fillColor, size));
```

Comment:
This will receive null as fillColor which is used as a param for `replace("$fill", fillColor)`. Maybe it should use a default color instead or is "null"/unrecognized color black or something like that?